AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginRESTIAMTests/RESTWithIAMIntegrationTests.swift

```diff
@@ -87,7 +87,6 @@ class RESTWithIAMIntegrationTests: XCTestCase {
     }
 
     func testGetAPIWithEncodedQueryParamsSuccess() async throws {
```

Comment:
we can enable these tests now in `.github/workflows/integ_test_api.yml` / `api-rest-iam-test` job / can remove `if: ${{ false }}`

Reply:
Addressed in [18a67b5](https://github.com/aws-amplify/amplify-swift/pull/2666/commits/18a67b550fa3c47b69c802778315f048c90993fc)